spec/unit_tests/stanford/storage_object_validator_spec.rb

```diff
@@ -8,14 +8,19 @@
     let(:verification_array) { storage_obj_validator1.validation_errors }
```

Comment:
it appears that the `let` statements in this outermost `describe` block aren't actually used?  vars of the same name are used later, but they also seem to be redefined later, within the narrower scopes that use them.